src/components/uikit/layout/Grid.tsx

```diff
@@ -17,7 +17,7 @@ export function Grid({
   spacing = 'none',
   align,
   justify,
-  columns,
+  columns = 3,
```

Comment:
Or how does this component actually work? 🤔 If no `columns` is provided will it produce the columns automatically based on the size of the children?

Reply:
It originally did not work if columns were not provided because of `const colWidth =
    columns !== undefined && width !== -1 ? width / columns : undefined;` and the fact that we do not show children if `colWidth` is undefined.
 
So either we find a way to produce the columns automatically or we make the columns prop mandatory.